models/userDao.js

```diff
@@ -0,0 +1,47 @@
+const appDataSource = require('./dataSource')
+
+const getUserById = async(userId) => {
+    return await appDataSource.query(`
+    SELECT
+        id,
+        account_email,
+        name,
+        profile_image
+        kakao_id
+    FROM users
+    WHERE id=?`
+    , [userId]
+    );
+}
+
+const getUserByKakaoId = async(kakaoId) => {
+    return await appDataSource.query(`
+    SELECT
+        id,
+        account_email,
+        name,
+        profile_image
+        kakao_id
+    FROM users
+    WHERE kakao_id=?`
+    , [kakaoId]
+    );
+}
+
+const signUp = async (email, name, kakaoId, profileImage) => {
```

Comment:
dao안의 dataSource의 query에 대한 naming은 가급적 crud 와 관련되게 지어주세요!
```suggestion
const createUser = async (email, name, kakaoId, profileImage) => {
```